lib/python/rose/macro.py

```diff
@@ -578,6 +579,7 @@ def load_meta_macro_modules(meta_files, module_prefix=None):
         except Exception:
             rose.reporter.Reporter()(
                 MacroLoadError(meta_file, traceback.format_exc()))
+        sys.path.pop()
```

Comment:
I guess you are confident that you don't need to do this within a `finally` here?